custom_tvm_ops/tests/layer_norm_test.py

```diff
@@ -0,0 +1,202 @@
+import numpy
+import torch
+import unittest
+from collections import namedtuple
+import itertools
+import logging
+import tvm
+import os
+
+source_location = os.path.abspath(__file__)
+pytorch_tvm_location = os.path.join(source_location, "../../..")
+pytorch_tvm_location = os.path.abspath(pytorch_tvm_location)
+logger = logging.getLogger()
+lib_path=os.path.join(pytorch_tvm_location, \
+        "build/custom_tvm_ops/topi/libcustom_tvm_ops_topi.so")
+torch.ops.load_library(lib_path)
+lib_path=os.path.join(pytorch_tvm_location, \
+        "build/custom_tvm_ops/relay/libcustom_tvm_ops_relay.so")
+torch.ops.load_library(lib_path)
```

Comment:
I think the library can be linked into torch_tvm and installed.

Reply:
agreed we certainly don't want this kind of logic if we can avoid it

Reply:
So I did try to fix this but since I wanted to make some forward progress I punted it for future. The other fix that include this in torch_tvm was not something I wanted to go for, about which I responded in another comment to @bwasti.